packages/pn-personafisica-webapp/src/utility/__test__/contacts.utility.test.ts

```diff
@@ -269,4 +271,10 @@ describe('Contacts utility test', () => {
 
     expect(result).toStrictEqual(currentAddresses);
   });
+
+  it('test sortAddresses function', () => {
```

Comment:
Write more tests cases:
- test order in legal addresses (PEC and SERCQ) -> test with one default and other not default and with both not default
- test order in courtesy addresses (IO, SMS and EMAIL)
- test order between legal and courtesy (i.e. PEC with SMS, SERCQ with EMAIL)

TIPS: create specific mock inside the single test case